docs/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM squidfunk/mkdocs-material:latest as build
```

Comment:
something for later we could improve is running it with less privileges, a ticket was created for us to track it https://github.com/kubefirst/kubefirst/issues/307

Reply:
never mind, I just saw the build process, and we dont use this image. closing #307